### ISSUE TITLE
Rendering Issues

### ISSUE THREAD

**Issue by ExcitedName** [phan-kavin/Quick-Consume#7]
Minecraft: 1.12.2
Forge: 14.23.5.2768
QuickConsume: 1.12.2.1.0.1.0
JEI: 4.14.4.264

I have noticed issues rendering the player skin in survival. And an issue with enchanted items shown in JEI. It doesn't seem to affect anything in the creative menus.

Survival - without JEI:
![2019-02-12_17 38 31](https://user-images.githubusercontent.com/17900146/52680708-51253d80-2eee-11e9-9451-6d20c2dbdd06.png)

Survival - with JEI:
![2019-02-12_17 40 31](https://user-images.githubusercontent.com/17900146/52680723-61d5b380-2eee-11e9-8acb-649acd6f8140.png)

**Comment by chillcat222** [comment]
I have the same issue, Its definitely quick consume

**Comment by Flammable4444** [comment]
Yeah, same problem here, it renders ok only if crafting book is oppened, otherwise it's like this.
